### PR TITLE
Sharon manager updates

### LINKED ISSUE #516: Manager Dashboard: Only show in progress appeals under "Current"
As a manager, I want to see what the caseworkers I am managing are working on. This list shouldn't be long, so there is no need for a search bar.

### Acceptance criteria
- Verify that all appeals that are currently in progress are displayed in the list under "Current"
- Verify that the name of the caseworker assigned to each task is displayed in the "Worked By" column
- Verify that there is no search bar associated with that list.

### Current state
screen shot 2016-12-02 at 2 47 40 pm

### Sub-tasks

- [ ] Start saving first and last names of users logging in. (database structure change)
- [ ] QA Manual testing

### PR DESCRIPTION
**Pull request by sharonwarner**
This PR updates the user database table to include the user's full name. It then displays the full name on the manager page. 

Fixes #516 

- [ ] Existing unit tests pass
- [ ] Added additional unit tests
- [ ] Confirmed in the UI

screen shot 2016-12-12 at 1 44 48 pm

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -70,6 +70,7 @@ def from_session(session)
       return nil if user.nil?
 
       find_or_create_by(css_id: user["id"], station_id: user["station_id"]).tap do |u|
+        u.full_name = user["name"]
```
Comment (joofsh) [inline app/models/user.rb]: Do you know if we save anywhere else downstream of this?  So if we hit the `create_by` part, and are creating a new user, we will successfully save their full_name on creation. 

However, if the user already exists and we hit the `find_` part, we are then only saving their full_name in memory, unless we save somewhere else in the code. 

I'm not sure the best way to handle this. On the one hand we may only want to save if the full_name was previously nil? On the other, the more frequently we save, the more likely the name in our system is up to date with whatever is in CSS (if someone gets married and changes their name). What do you think?

Comment (sharonwarner) [inline app/models/user.rb]: Ah, makes sense. Seems like it wouldn't be too hard to just save every time the user logs in. I'll update that.

File: app/models/user.rb [hunk 2]
```diff
@@ -70,8 +70,10 @@ def from_session(session)
...
         u.roles = user["roles"]
         u.regional_office = session[:regional_office]
+        u.save
```
Comment (shanear) [inline app/models/user.rb]: we could avoid a second DB call on most requests (most requests wont be a user's first ever request) if we only do the second save when something has changed.

It's possible that #railsmagic :sparkles: will also take care of avoiding the call. Do you want to research that?

File: lib/fakes/initializer.rb [hunk 3]
```diff
@@ -9,7 +9,8 @@ def self.development!
     User.authentication_service.user_session = {
       "id" => "ANNE MERICA",
       "roles" => ["Certify Appeal", "Establish Claim", "Manage Claim Establishment"],
-      "station_id" => "283"
+      "station_id" => "283",
+      "full_name" => "Anne Merica"
```
Comment (shanear) [inline lib/fakes/initializer.rb]: i believe Anne Merica's name should be spelled in all caps with 2 exclaimation points, like this: ANNE MERICA!!

jk jk :smile:

**Pull request merged**